doc/install_script.sh

```diff
@@ -1072,7 +1072,7 @@ then
     # Install setuptools
     do_setup_py $SETUPTOOLS
 
-    if type -p git &>/dev/null
+    if type -P git &>/dev/null
```

Comment:
I changed it to match the other invocations of `type` in the script, just for consistency. I don't think it actually matters.